templates/content-page/reports/a11y-1.json

```diff
@@ -0,0 +1,638 @@
+        "earl:subject": "_:subject",
+        "earl:outcome": "earl:inapplicable",
+        "dct:description": "",
+        "earl:mode": "earl:semiAuto",
+        "@type": ["earl:TestResult", "earl:Assertion"]
+      },
+      {
+          "earl:test": "WCAG22:visible-controls",
+        "earl:subject": "_:subject",
+        "earl:outcome": "earl:cantTell",
+        "dct:description": "",
+        "earl:mode": "earl:semiAuto",
+        "@type": ["earl:TestResult", "earl:Assertion"]
+      },
+      {
+        "@id": "_:sc_3.3.1",
+        "earl:test": "WCAG21:error-identification",
+        "earl:subject": "_:subject",
+        "earl:outcome": "earl:inapplicable",
+        "dct:description": "",
+        "earl:mode": "earl:semiAuto",
+        "@type": ["earl:TestResult", "earl:Assertion"]
+      },
+      {
+        "@id": "_:sc_3.3.2",
+        "earl:test": "WCAG21:labels-or-instructions",
+        "earl:subject": "_:subject",
+        "earl:outcome": "earl:cantTell",
+        "dct:description": "",
+        "earl:mode": "earl:semiAuto",
+        "@type": ["earl:TestResult", "earl:Assertion"]
+      },
+      {
+        "@id": "_:sc_3.3.3",
+        "earl:test": "WCAG21:error-suggestion",
+        "earl:subject": "_:subject",
+        "earl:outcome": "earl:inapplicable",
+        "dct:description": "",
+        "earl:mode": "earl:semiAuto",
+        "@type": ["earl:TestResult", "earl:Assertion"]
+      },
+      {
+        "@id": "_:sc_3.3.4",
+        "earl:test": "WCAG21:error-prevention-legal-financial-data)",
+        "earl:subject": "_:subject",
+        "earl:outcome": "earl:inapplicable",
+        "dct:description": "",
+        "earl:mode": "earl:semiAuto",
+        "@type": ["earl:TestResult", "earl:Assertion"]
+      },
+      {
+        "earl:test": "WCAG22:accessible-authentication",
+        "earl:subject": "_:subject",
+        "earl:outcome": "earl:inapplicable",
+        "dct:description": "",
+        "earl:mode": "earl:semiAuto",
+        "@type": ["earl:TestResult", "earl:Assertion"]
+      },
+      {
+        "earl:test": "WCAG22:redundant-entry",
+        "earl:subject": "_:subject",
+        "earl:outcome": "earl:inapplicable",
+        "dct:description": "",
+        "earl:mode": "earl:semiAuto",
+        "@type": ["earl:TestResult", "earl:Assertion"]
+      },
+      {
+        "@id": "_:sc_4.1.1",
+        "earl:test": "WCAG21:parsing",
+        "earl:subject": "_:subject",
+        "earl:outcome": "earl:cantTell",
+        "dct:description": "",
+        "earl:mode": "earl:semiAuto",
+        "@type": ["earl:TestResult", "earl:Assertion"]
+      },
+      {
+        "@id": "_:sc_4.1.2",
+        "earl:test": "WCAG21:name-role-value",
+        "earl:subject": "_:subject",
+        "earl:outcome": "earl:cantTell",
+        "dct:description": "",
+        "earl:mode": "earl:semiAuto",
+        "@type": ["earl:TestResult", "earl:Assertion"]
+      },
+      {
+            "acr:asset": {
```

Comment:
This attachment is just floating, I think it goes into the `_:sc_4.1.2`

Set at the description the text in the image which is : Missing value for 'content' attribute. If either 'name', 'http-equiv', 'property', or 'itemprop' is specified, then the 'content' attribute must have a value